src/TLuaInterpreter.cpp

```diff
@@ -5182,6 +5182,34 @@ int TLuaInterpreter::getMudletHomeDir( lua_State * L )
     return 1;
 }
 
+// returns search paths for LuaGlobal itself to look at when loading other modules
+// follows the principle of closest paths to the binary first, furthest away lasy
+int TLuaInterpreter::getMudletLuaDefaultPaths( lua_State * L )
+{
+    int index = 1;
+    lua_newtable( L );
+#if defined(Q_OS_MAC)
+    lua_createtable(L,3,0);
+#else
+    lua_createtable(L,2,0);
```

Comment:
I'm not a Lua genius, so is a comment about why there is a difference here for Macs appropriate?